frontend/lib/routes/meeting/MeetingCard.dart

```diff
@@ -0,0 +1,129 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:frontend/models/meeting.dart';
+
+class MeetingCard extends StatelessWidget {
+  //final Meeting meeting;
+  final List months = [
+    'JAN',
+    'FEB',
+    'MAR',
+    'APR',
+    'MAY',
+    'JUN',
+    'JUL',
+    'AUG',
+    'SEP',
+    'OCT',
+    'NOV',
+    'DEC'
+  ];
+
+  MeetingCard({
+    Key? key,
+    /*required this.meeeting*/
+  }) : super(key: key);
+
+  String getTime(DateTime time) {
```

Comment:
Take a look at [this package](https://pub.dev/packages/intl), might help you simplify dates to string

Reply:
nice, already changed it